Classes/UT3Paladin.uc

```diff
@@ -49,181 +49,225 @@
-    Wheels(3) = RWheel4;
-    Wheels(4) = LWheel1;
-    Wheels(5) = LWheel2;
-    Wheels(6) = LWheel3;
-    Wheels(7) = LWheel4;
-    // @100GPing100
-    //======END======
-
-    VehicleNameString = "UT3 Paladin" //GE: UT3 Paladin...
-    SteerSpeed=90                     //Is steered more easily
-    ChassisTorqueScale=0.1            //Has a lower... setting of some kind
-    MaxBrakeTorque=75.000000          //And another, but this time higher
-    MaxSteerAngleCurve=(Points=((OutVal=20.000000),(InVal=700.000000,OutVal=15.000000)))//Again steered more easily
-    //EngineBrakeFactor=0.100000       //This makes it extremely easy to flip and have an accident
-    WheelInertia=0.750000             //Has more inertia
-    //GroundSpeed=1000.000000           //Is faster, GroundSpeed == deprecated?
-    //DriverWeapons(0)=(WeaponClass=class'UT3PaladinCannon',WeaponBone=CannonAttach);//Has a better shield
+    Wheels(0)=RWheel1;
+    Wheels(1)=RWheel2;
+    Wheels(2)=RWheel3;
+    Wheels(3)=RWheel4;
+    Wheels(4)=LWheel1;
+    Wheels(5)=LWheel2;
+    Wheels(6)=LWheel3;
+    Wheels(7)=LWheel4;
+
+    VehicleNameString="UT3 Paladin"   //GE: UT3 Paladin...
+    ImpactDamageMult=0.0003
+    SteerSpeed=82                       // Is steered more easily
+    ChassisTorqueScale=0.18             // Has a lower... setting of some kind
+    MinBrakeFriction=4.2
+    MaxBrakeTorque=75.000000            // And another, but this time higher
+    MaxSteerAngleCurve=(Points=((OutVal=20.000000),(InVal=700.000000,OutVal=15.000000))) // Again, steered more easily
+//    EngineBrakeFactor=0.100000        // This makes it extremely easy to flip and have an accident
+    WheelInertia=0.750000               // Has more inertia
+    
+    GroundSpeed=700.000000              //Is faster, GroundSpeed == deprecated?
+    //DriverWeapons(0)=(WeaponClass=class'UT3PaladinCannon',WeaponBone=CannonAttach); // Has a better shield
+    
+    TurnDamping=25     //35
+    TransRatio=0.030    //0.03
+    ChangeUpPoint=2600 //2000
+    
+    MomentumMult=0.1 //0.8
+    
+    ExitPositions(0)=(X=0,Y=-250,Z=30)
+    ExitPositions(1)=(X=0,Y=250,Z=30)
+    ExitPositions(2)=(X=0,Y=-250,Z=-30)
+    ExitPositions(3)=(X=0,Y=250,Z=-30)
+    
+    Begin Object Class=KarmaParamsRBFull Name=KParams0
+        KStartEnabled=True
+        KFriction=0.5
+        KLinearDamping=0.05
+        KAngularDamping=0.05
+        KImpactThreshold=700
+        KMaxSpeed=1200
+        bKNonSphericalInertia=True
+        bHighDetailOnly=False
+        bClientOnly=False
+        bKDoubleTickRate=True
+        KInertiaTensor(0)=1.0
+        KInertiaTensor(1)=0.0
+        KInertiaTensor(2)=0.0
+        KInertiaTensor(3)=3.0
+        KInertiaTensor(4)=0.0
+        KInertiaTensor(5)=3.0
+        KCOMOffset=(X=-0.25,Y=0.0,Z=-1.35)
+        bDestroyOnWorldPenetrate=True
+        bDoSafetime=True
+        Name="KParams0"
+    End Object
+    KParams=KarmaParams'KParams0'
+    
+    HeadlightCoronaOffset=()
```

Comment:
This seems unnecessary, as far as I can see there are no headlight coronas in ONSShockTank to begin with.